CONTRIBUTING.md

```diff
@@ -24,15 +24,16 @@ The submitted code should adhere to the following:
 
 ### Do's
 
-* For every pull request, make sure all commits are consolidated into one commit before
-  sending the pull request.
+* Keep pull requests focused on addressing one issue or feature. If multiple separate issues
```

Comment:
PR's when merged will always be squashed. We shouldn't be having multiple fixes in one pull request or break larger ones into smaller, separate PR's.

> If multiple separate issues... prefer commiting the fixes under multiple commits, one for each issue

should be removed.